src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -33,10 +45,21 @@ def fdb_vlanmac(self, fdb):
                     vlan_id = vlan_id.decode()
                 self.bvid_vlan_map[fdb["bvid"]] = vlan_id
         else:
-            return None
+            return None, None
         if not isinstance(vlan_id, str):
-            return None
-        return (int(vlan_id),) + mac_decimals(fdb["mac"])
+            return None, None 
+        return (int(vlan_id),) + mac_decimals(fdb["mac"]), int(vlan_id)
```

Comment:
>int(vlan_id) [](http://example.com/codeflow?start=59&length=12)

Since normally it is the head element in original return tuple. No need to add extra return value.

Reply:
I will check and change